docs/standard/serialization/binaryformatter-security-guide.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "BinaryFormatter security guide"
+description: This article describes the security risks inherent in the BinaryFormatter type and recommendations for different serializers to use.
+ms.date: "07/11/2020"
+helpviewer_keywords: 
+ms.assetid: 00000000-0000-0000-0000-000000000000
+author: "GrabYourPitchforks"
+---
+# BinaryFormatter security guide
+
+This document applies to the following framework versions:
+
+* .NET Framework (all versions)
+* .NET Core 2.1 - 3.1
+* .NET 5.0+
+
+## Background
+
+> __Summary__: The `BinaryFormatter` type is dangerous and is not recommended for data processing. Applications should stop using it at their earliest opportunity, even if they believe the data they're processing to be trustworthy.
```

Comment:
```suggestion
> __Summary__: The <xref:System.Runtime.Serialization.Formatters.Binary.BinaryFormatter> type is dangerous and is not recommended for data processing. Applications should stop using it at their earliest opportunity, even if they believe the data they're processing to be trustworthy.
```

Reply:
> Applications should stop using it at their earliest opportunity, even if they believe the data they're processing to be trustworthy.

As a normal reader, this is confusing for me, why would a trusted source be a problem?
If this is meant that I *may* think it's trusted while it's not, I'd indicate that clearly.